ui/components/app/advanced-gas-controls/advanced-gas-controls.component.js

```diff
@@ -29,23 +45,31 @@ export default function AdvancedGasControls({
           onManualChange?.();
           setGasLimit(value);
         }}
-        tooltipText={t('editGasLimitTooltip')}
-        value={gasLimit}
-        allowDecimals={false}
-        numeric
       />
       <>
-        <FormField
-          titleText={t('advancedGasPriceTitle')}
-          titleUnit="(GWEI)"
```

Comment:
how can we add `titleUnit` with label component?

Reply:
Hey @dhruvv173, you may want to checkout this story and check the code for it https://metamask.github.io/metamask-storybook/?path=/story/components-componentlibrary-formtextfield--custom-label-or-help-text

Reply:
I referred to the same story and docs while working on this. But the current component displays unit as shown in the image below, still trying to figure out how we can add that, will look into it in the morning, thank you for your help!
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/07f02f13-a7df-4ee8-8c5a-902b65bd221b)

Reply:
completely forgot about the `Text` component which could be used to display the unit, thanks for the update!